components/generic/CookieDisclaimer.vue

```diff
@@ -61,6 +63,10 @@
 
       if (status !== 'accepted') {
         this.isOpen = true;
+        this.accepted = false;
+      }
+      if (status === 'accepted') {
```

Comment:
Turned it into the following to avoid negated condition.
```javascript
if (status === 'accepted') { 
    this.accepted = true; 
} else { 
    this.isOpen = true; 
    this.accepted = false; 
}
```